stdlib/public/Darwin/Foundation/NSSet.swift

```diff
@@ -77,10 +77,8 @@ extension Set : _ObjectiveCBridgeable {
       // Swift. See rdar://problem/35995647
       var set = Set(minimumCapacity: s.count)
       s.enumerateObjects({ (anyMember: Any, _) in
-        let member = Swift._forceBridgeFromObjectiveC(
-          anyMember as AnyObject, Element.self)
         // FIXME: Log a warning if `member` is already in the set.
-        set.insert(member)
+        set.insert(anyMember as AnyObject as! Element)
```

Comment:
Is the double cast still necessary here?

Reply:
Good point -- I don't think we need it.